docs/modules/gear/items.mdx

```diff
@@ -759,6 +758,160 @@ Preferably books should be written in-game to ensure proper formatting, and then
 </book>
 ```
 
+### Firework
+
+You can define custom explosion designs when creating and giving a firework rocket item.
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Elements</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>{`<firework> </firework>`}</label>
+        </td>
+        <td>A firework design containing one or more explosion effect.</td>
+      </tr>
+      <tr>
+        <th>Sub-elements</th>
+        <th />
+      </tr>
+      <tr>
+        <td>
+          <label>{`<explosion> </explosion>`}</label>
+        </td>
+        <td>
+          Define an explosion effect design.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+##### Explosion Attributes
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Attribute</th>
+        <th>Description</th>
+        <th>Value</th>
+        <th>Default</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>type</label>
+        </td>
+        <td>
+          Define the type of explosion effect.
+        </td>
+        <td>
+          <label>ball</label>,
+          <label>ball large</label>, <br />
```

Comment:
```suggestion
          <label>ball large</label>
```
This `<br>` tag might be unnecessary.

Reply:
It was added to keep the size of one of the table column from extending past the page view area.